packages/src/back_to_school_josh/utils.py

```diff
@@ -0,0 +1,30 @@
+__all__ = ["unwrap", "T"]
+
+from collections.abc import Iterable
+from typing import (
+    TypeVar,
+)
+
+T = TypeVar("T")
+
+
+def unwrap(container: Iterable[T], msg: str = "") -> T:
```

Comment:
Do you actually use unwrap anywhere? 😅

Reply:
I'm not sure, its definitely possible I don't in this repo, but I almost certainly will at some point - I find it so useful that its always handy to have it around, kinda wish it was a builtin.